biojava-structure/src/main/java/org/biojava/nbio/structure/asa/AsaCalculator.java

```diff
@@ -293,10 +325,22 @@ public AsaCalculator(Point3d[] atomCoords, double probe, int nSpherePoints, int
 			while (!threadPool.isTerminated());
 
 		}
+		end = System.currentTimeMillis();
+		logger.debug("Took {} s to calculate all {} atoms ASAs (excluding neighbors calculation)", (end-start)/1000.0, atomCoords.length);
 
 		return asas;
 	}
 
+	/**
+	 * Set the useSpatialHashingForNeighbors flag to use spatial hashing to calculate neighbors (true) or all-to-all
+	 * distance calculation (false). Default is {@value DEFAULT_USE_SPATIAL_HASHING}.
+	 * Use for testing performance only.
```

Comment:
For small molecules is the old algorithm faster? If so maybe document that here.

Reply:
Performance is essentially the same for small molecules. The gain is very noticeable in very large molecules.